pkg/verifier/aws/aws_verifier.go

```diff
@@ -276,23 +278,14 @@ func (a *AwsVerifier) findUnreachableEndpoints(ctx context.Context, instanceID s
 				return true, nil
 			}
 
-			// Check consoleOutput for failures, report as exceptions if they occurred
-			genericFailures := reGenericFailure.FindAllStringSubmatch(consoleLogs, -1)
-			if len(genericFailures) > 0 {
-				a.writeDebugLogs(fmt.Sprint(genericFailures))
-
-				dockerFailures := reDockerFailure.FindAllString(consoleLogs, -1)
-				if len(dockerFailures) > 0 {
-					// Should be resolved by OSD-13003 and OSD-13007
-					a.Output.AddException(handledErrors.NewGenericError(errors.New("docker was unable to install or run. Further investigation needed")))
-					a.Output.AddError(handledErrors.NewGenericError(fmt.Errorf("%v", dockerFailures)))
-				} else {
-					// TODO: Flesh out generic issues, for now we only know about Docker
-					a.Output.AddException(handledErrors.NewGenericError(errors.New("egress tests were not run due to an uncaught error in setup or execution. Further investigation needed")))
-					a.Output.AddError(handledErrors.NewGenericError(fmt.Errorf("%v", genericFailures)))
-				}
+			// Add a message to debug logs if we're using the prepulled image
+			prepulledImage := rePrepulledImage.FindAllString(consoleLogs, -1)
+			if len(prepulledImage) > 0 {
+				a.writeDebugLogs(prepulledImageMessage)
 			}
 
+			a.isGenericErrorPresent(consoleLogs)
```

Comment:
Added this debug log when this is true:
```
"generic error found - please help us classify this by sharing it with us so that we can provide a more specific error message"
```

Reply:
yup, looks fine 👍